websites/A/AnizmTV/dist/metadata.json

```
@@ -1,8 +1,8 @@
 {
 	"$schema": "https://schemas.premid.app/metadata/1.7",
 	"author": {
-		"name": "Bas1C",
-		"id": "303603391941050370"
+		"name": "Bashiku#6817",
```

it's the name not the tag
```suggestion
		"name": "Bashiku",
```